Tests/CommandsTests/TestCommandTests.swift

```diff
@@ -1017,7 +1013,7 @@ struct TestCommandTests {
                 )
             }
         } when: {
-            buildSystem == .swiftbuild
+            buildSystem == .swiftbuild && ProcessInfo.hostOperatingSystem != .macOS
```

Comment:
issue (possibly blocking): Do we know why this test failed on Linux and Windows?  Could we add an issue trait to the test to track it?